src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -402,4 +402,27 @@ Mmr.propTypes = {
   number: PropTypes.number,
 };
 
+export const CompetitiveRank = ({ rankTier }) => (
+  <span>
+    <section
+      data-hint="Competitive rank"
+      data-hint-position="bottom"
+    >
+      <SocialPerson />
+    </section>
+    {rankTier && rankTier === parseInt(rankTier, 10) ?
+      <span>
+        {strings[`rank_tier_${parseInt(rankTier / 10, 10)}`]} { (parseInt(rankTier, 10) > 9) && <span>[{parseInt(rankTier % 10, 10)}]</span> }
```

Comment:
We may want to move this logic to a utility function (we also use it for rank distribution, should use the shared function there)